### DISCUSSION TITLE
Are you going to add air window natively in the next update? I feel that it is a very powerful extension that would improve Cardinal a lot more.

### DISCUSSION THREAD

**Discussion by yanyoart** [DISTRHO/Cardinal#691] [Ideas]
duda

**Comment by dromer** [comment]
Unlikely.

It has already been requested here: https://github.com/DISTRHO/Cardinal/discussions/63#discussioncomment-8202538
And here: https://github.com/DISTRHO/Cardinal/issues/488

And module requests go here: https://github.com/DISTRHO/Cardinal/wiki/Possible-modules-to-include
